intelmq/lib/bot.py

```diff
@@ -813,7 +813,7 @@ def process(self):
                 else:
                     events = [value]
             except Exception:
-                self.logger.exception('Failed to parse line.')
+                self.logger.exception("Failed to parse line '{}' for feed '{}'".format(line, report['feed.url']))
```

Comment:
You can't rely on the existence of `feed.url`.

Reply:
Thanks for that. I fixed the issue in the new commit